Misc/NEWS.d/next/Core and Builtins/2019-12-17-22-32-11.bpo-13601.vNP4LC.rst

```diff
@@ -0,0 +1,2 @@
+``sys.stderr`` is always line-buffered now, even if ``stderr`` is
```

Comment:
Unless it is unbuffered.

Reply:
This is your first contribution. Please add your name in `Misc/ACKS`. You can also add "Patch by *yourname*." here.

Reply:
Thanks, I added some explanations.